server/lib/directory.coffee

```diff
@@ -0,0 +1,30 @@
+path = require 'path'
+fs = require 'fs'
+root = '/usr/local/var/cozy'
+spawn = require('child_process').spawn
+
+###
+    Create directory for <app>
+###
+module.exports.create = (app, callback) ->
+    changeOwner = (path, callback) ->
```

Comment:
@poupotte if you can move this function into another function at the root level of the module, it would be cleaner.